src/proc/rotation-filter.cpp

```diff
@@ -52,32 +53,51 @@ namespace librealsense {
         register_option( RS2_OPTION_ROTATION, rotation_control ); 
     }
 
+    rotation_filter::rotation_filter( std::vector< rs2_stream > streams_to_rotate )
```

Comment:
Code duplication. Is default constructor needed? If so have it call the other constructor with an empty vector `{}`.